doc/customize.md

```diff
@@ -0,0 +1,449 @@
+hopefully the current default behaviour is just fine. We might remove
+`auto_select_fragment` as a setting altogether; we might also turn
+`auto_select` into a mere boolean, since the current setting
+`auto_select = "first"` has not proved of any practical value.
+Regardless, it seems like the most meaningful combinations as of now
+are either `auto_select = "none"` - in which case the other setting is
+ignored, or `auto_select = "code"` and `auto_select_fragment = true`,
+which now is the default.
+
+### Change the width and height of slides
+
+To control the width and height of your slides, use the following
+configuration:
+
+    {
+     ...
+     "rise": {"width": "90%",
+              "height": "90%"}
+    }
+
+**Important notes**
+
+* remember that you can always use your browser's shortcuts to zoom
+in/out (`Cmd/Ctrl +` and `Cmd/Ctrl -`), and this way adjust the slide
+content to your screen/projector size.
+
+* this method is *often preferable* than setting sizes. In particular
+  it is dangerous to set sizes in pixels, as most often you cannot
+  rehearse with the actual projector. We recommend setting relative
+  sizes (in percents) rather than absolute ones (in `px` or `cm`).
+
+* in any case you may want to increase the slide height to ensure that
+cell outputs fit within a single slide; keep in mind that cell
+contents tend to take more space as you run your code.
+
+### Decorating all slides
+
+RISE offers two levels for inserting a static background. You can either
+
+* define `overlay`, in which case you take full control,
+* **or** you can define `header`, `footer` and `backimage`.
+
+So if you define `overlay`, the 3 latter options will be ignored.
+
+
+#### `overlay`
+
+It is possible to add the config option `overlay` to build a constant
+background.  It is wrapped in a`<div>`, so it can be text or html.  In
+this case, the user is entirely responsible for styling.  For example:
+
+    {
+     ...
+     "rise": {
+         "overlay": "<div class='myheader'><h2>my company</h2></div><div class='myfooter'><h2>the date</h2></div>"
+     }
+    }
+
+#### `header`, `footer` and `backimage`
+
+As a more limited, but often more convenient alternative, you can
+define any of the following 3 settings.
+
+In this case, minimal styling is applied (floor and ceiling), but user
+is still responsible for cosmetic styling:
+
+    {
+     ...
+     "rise": {
+         "backimage": "mybackimage.png",
+         "header": "<h1>Hello</h1>",
+         "footer": "<h3>World!</h3>"
+     }
+    }
+
+You can see some examples using these options at
+`RISE/examples/overlay.ipynb` and
+`RISE/examples/header-footer.ipynb``, or in binder respectively
+[![](https://mybinder.org/badge.svg)](https://mybinder.org/v2/gh/parmentelat/RISE.git/doc2?filepath=examples%2Foverlay.ipynb)
+[![](https://mybinder.org/badge.svg)](https://mybinder.org/v2/gh/parmentelat/RISE.git/doc2?filepath=examples%2Fheader-footer.ipynb)
+
+
+### `toolbar_icon`
```

Comment:
Are we sure we want to expose this? I think we should use a canonical icon (the one we are using right now) to avoid confusion with other extensions.
Maybe keep the config availability to change the icon but get rid of it in the docs... so if someone **really** need to change the icon, they can still do it.

Reply:
fine - will be in today's commit

Reply:
Thx!